JSON-connect/package-lock.json

```diff
@@ -0,0 +1,7360 @@
+{
```

Comment:
Could you remove this? we're using yarn for the other repos so we should have yarn.lock's instead of package-lock.json.

Reply:
Sorry I missed this on the first look-over.

Reply:
done